LEGO1/omni/src/video/mxdisplaysurface.cpp

```diff
@@ -1397,7 +1401,7 @@ void MxDisplaySurface::VTable0x2c(
 }
 
 // FUNCTION: LEGO1 0x100bc8b0
-LPDIRECTDRAWSURFACE MxDisplaySurface::FUN_100bc8b0(MxS32 width, MxS32 height)
+LPDIRECTDRAWSURFACE MxDisplaySurface::FUN_100bc8b0(MxS32 p_width, MxS32 p_height)
```

Comment:
Can you submit this change to the decomp repo as well? Thank you

Reply:
Only did it to have the build not fail ...

Reply:
Yes, I figured. I'm not sure why the build isn't failing in the decomp. But anything that isn't related to the porting effort we should ideally "back port" to the decomp, to make merging updates easier

Reply:
https://github.com/isledecomp/isle/pull/1472